src/operator/tensor/elemwise_binary_op_extended.cc

```diff
@@ -26,9 +26,8 @@
 
 namespace mxnet {
 namespace op {
-MXNET_OPERATOR_REGISTER_BINARY(_power)
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(_power, mshadow_op::power)
```

Comment:
Add where?  You marked code for the normal binary power() function which won't have a scalar attribute to check.